networking/ovn_kubernetes_network_provider/ovn-k-network-policy.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ovn-k-network-policy"]
+= OVN-Kubernetes network policy
+include::_attributes/common-attributes.adoc[]
+:context: ovn-k-network-policy
+
+toc::[]
+
+:FeatureName: `AdminNetworkPolicy`
+include::snippets/technology-preview.adoc[]
+
+Kubernetes offers two features that users can use to enforce network security. One feature that allows users to enforce network policy is the `NetworkPolicy` API that is designed for application developers and namespace tenants. For more information, see xref:../../networking/network_policy/about-network-policy.adoc#nw-networkpolicy-about_about-network-policy[About network policy].
+
+The second feature is `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP) APIs. Cluster administrators can use ANP and BANP to enforce non-overridable policies that take precedence over `NetworkPolicies`.
```

Comment:
Maybe?
```suggestion
The second feature is `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP) APIs. Cluster administrators can use ANPs and BANPs to enforce non-overridable policies that take precedence over `NetworkPolicies`.
```

Reply:
This is tricky situation with plurals. You can only have one BANP so I don't think that can ever be plural. I'm going leave singular for clarity.